src/public/log/schema.rs

```diff
@@ -548,9 +550,9 @@ pub enum ObservableTypeId {
     IPAddress = 2,
     /// Media Access Control (MAC) address. For example: 18:36:F3:98:4F:9A
     MACAddress = 3,
-    /// User name. For example: john_doe
+    /// User name. For example: `john_doe`
     UserName = 4,
-    /// Email address. For example: john_doe@example.com
+    /// Email address. For example: `john_doe@example.com`
```

Comment:
I don't think this needs to be in back ticks. Same for the user name since it's not code

Reply:
I don't know why but it errors if I don't 😭, I'll remove it, push it, and see what happens

Reply:
Oh, it needs to be escaped